vowpalwabbit/cb_explore_adf_cover.cc

```diff
@@ -282,7 +282,10 @@ VW::LEARNER::base_learner* setup(config::options_i& options, vw& all)
   }
 
   // Set explore_type
-  const size_t problem_multiplier = cover_size + 1;
+  size_t problem_multiplier = cover_size + 1;
+
+  // Cover is using doubly robust without the cooperation of the base reduction
+  if (cb_type_enum == CB_TYPE_MTR) { problem_multiplier *= 2; }
```

Comment:
Did this come from somewhere?

Reply:
what do you mean with that question, can you rephrase?

Reply:
Sorry, it was unclear. This is a new line. Can you explain why it was added?

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/blob/70d4863d382bf2dda53918db7e17b069498b86b4/vowpalwabbit/cb_explore_adf_cover.cc#L273

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/blob/70d4863d382bf2dda53918db7e17b069498b86b4/vowpalwabbit/cb_adf.cc#L500

Reply:
mtr is using DR for some policies. cb_adf does the correct thing on adjusting when using dr but at that point cb_adf registers as using "mtr" skipping the dr adjustment. This syncs that up. See both files linked above this comment for more context.